CONTRIBUTING.md

```diff
@@ -0,0 +1,174 @@
+
+### Troubleshooting
+
+1. You can face with pod failing with `Init:CrashLoopBackOff` issue. Once you get logs by running `kubectl logs pxc-cluster-pxc-0 -c pxc-init` you get the error `install: cannot create regular file '/var/lib/mysql/pxc-entrypoint.sh': Permission denied`. You can fix it using [this solution](https://github.com/kubernetes/minikube/issues/12360#issuecomment-1123794143). Also, check [this issue](https://jira.percona.com/browse/K8SPXC-879)
+2. Multinode PXC Cluster can't be created on ARM CPUs. You can have single node installation.
+3. Operators are not supported. It means that the PMM version <-> operator version pair does not exist in the Version service. This issue can happen in two different scenarios. You can have a PMM version higher than the current release, or you installed a higher version of operators. You can check compatibility using https://check.percona.com/versions/v1/pmm-server/PMM-version
+
+## Submitting a Pull Request
+
+See [Working with Git and GitHub](docs/process/GIT_AND_GITHUB.md) and [Common tech libraries and approaches](docs/process/common_tech_lib_approaches.md)
+
+As a PR created you are responsible to:
+* make sure PR is ready (linted, tested and etc)
+* make sure it is reviewed (ask for review, ping reviewers and etc)
+* make sure it is merged
+  * merge when it is reviewed and tested
+  * ask code owners/admins to merge it if merging is blocked for some reason
+
+## Feature Build
+
+PMM is quite complex project, it consists from many different repos descibed above. Feature Build (FB) is a way to get changes all together, build them all together, run tests and get client and server containers.
+
+Please see: [How to create a feature build](https://github.com/Percona-Lab/pmm-submodules/blob/PMM-2.0/README.md#how-to-create-a-feature-build)
+
+### The Goals of the Feature Builds
+
+1. Provide an easy way to test/accept functionality for PO/PM and QA
+2. Inform the Developer about Automation Tests results before the code is merged
+3. (Future) Let the Developers add/modify e2e tests when they change functionality
+
+### The Rules
+
+1. Start Feature Build for every feature/improvement you are working on.
+2. Start PullRequest to percona-lab/pmm-submodules as DRAFT.
+3. Change the status of Pull Request from Draft to Open ONLY if your changes must be merged to pmm-submodules.
+4. Include a short explanation in the Long Description field of the Feature in PR for feature build and checkboxes to all related Pull Requests. Check other [PRs](https://github.com/Percona-Lab/pmm-submodules/pulls) as examples.
+5. After all related PRs in feature build are merged you should:
+   a. either close the PR and delete the branch (this is the default option) or
+   b. merge the PR to pmm-submodules repository (please note, this rarely needs to be merged, for example infrastructure changes do)
```

Comment:
do we need to duplicate these parts? isn't it better to just point them to pmm docs? Having them in both places makes it harder to manage.

Reply:
I think that we'll move to GitHub for the developer documentation. I agree that it makes harder to manage but it's more friendly for newbies.

Reply:
I didn't get you. PMM docs are in Github as well, so no links outside of GitHub.